src/platform-includes/performance/opentelemetry-install/go.mdx

```diff
@@ -0,0 +1,18 @@
+<Note>
+
+OpenTelemetry integration was added in `sentry-go` version 0.18.0, and works only for Go >= 1.18.
+
+</Note>
+
+<Note>
+
+The Sentry Go OpenTelemetry integration requires `go.opentelemetry.io/otel` (and its submodules) of version `1.11.0` or higher.
+
+</Note>
+
+Install the `otel` module in addition to the main SDK:
+
+```bash
+go get github.co/getsentry/sentry-go \
+       github.co/getsentry/sentry-go/otel
```

Comment:
```suggestion
go get github.co/getsentry/sentry-go@latest \
       github.co/getsentry/sentry-go/otel@latest
```

Maybe?

Reply:
Tbh, not exactly sure what is the difference?

Reply:
But `github.co` is :)